airbyte-integrations/connectors/source-google-search-console/unit_tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from pytest import fixture
+
+
+@fixture(name="config")
+def config_fixture(requests_mock):
+    url = "https://oauth2.googleapis.com/token"
+    requests_mock.post(url, json={"access_token": "token", "expires_in": 10})
```

Comment:
Is there a reason we mocking the response here and don't reuse it elsewhere?

Reply:
It used when we try to pass authorization.